src/test/resources/testbash/auxiliary_computational_requirements/check_computational_requirements.vsh.yaml

```diff
@@ -0,0 +1,35 @@
+functionality:
+  name: auxiliary_computational_requirements
+  version: 0.1
+  description: |
+    Check setting of computational requirements
+  resources:
+    - type: bash_script
+      text: |
+        if [ -n "$VIASH_META_CPUS" ]; then
+          echo "cpus: $VIASH_META_CPUS"
```

Comment:
could you use `$meta_cpus` instead? `$VIASH_META_CPUS` is not meant to be used by end users

Reply:
done!